doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>minor: some missing commas before 'and'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I don't think they are missing commas...</p>